<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FesParamBtn</class>
 <widget class="QWidget" name="FesParamBtn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>训练时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="data_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unit_Label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>title_Label</zorder>
  <zorder>data_Label</zorder>
  <zorder>unit_Label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
